heron/packing/src/java/com/twitter/heron/packing/roundrobin/RoundRobinPacking.java

```diff
@@ -144,11 +143,11 @@ public PackingPlan pack() {
           new PackingPlan.ContainerPlan(containerId, instancePlanMap, resource);
 
       containerPlanMap.put(containerId, containerPlan);
+      topologyRam += containerRam;
```

Comment:
Currently packing algorithm has no knowledge about TMaster ram requirement. As `AuroraScheduler` is requesting a container for container-0, it increments the topologyRam after receiving a packing plan. See new `AuroraScheduler.java`#L136 in the diff